tofhir-server/src/main/scala/io/tofhir/server/model/FhirMappingTaskTest.scala

```diff
@@ -0,0 +1,31 @@
+package io.tofhir.server.model
+
+import io.tofhir.engine.model.FhirMappingTask
+
+/**
+ * FhirMappingTask and selection of rows to test a mapping
+ * @param fhirMappingTask task to be used for testing
+ * @param selection row selection for testing
+ */
+case class FhirMappingTaskTest(fhirMappingTask: FhirMappingTask,
```

Comment:
Found `FhirMappingTaskTest` name misleading. Even, I would not have such a class but separate mapping task and `RowSelection` parameters. I would name RowSelection something like ResourceCriteria or ResourceFilter